app/classifier/tasks/dropdown/index.spec.js

```diff
@@ -81,6 +88,23 @@ describe('DropdownTask:static methods', function () {
 });
 
 describe('DropdownTask', function () {
+  describe('with single select', function () {
+    const annotation = { value: [{ value: 0, option: true }] };
+
+    const wrapper = mount(
```

Comment:
```suggestion
    const wrapper = shallow(
```
I'm switching this to shallow rendering because we're testing that the correct value gets passed to Select, but we don't really care how Select renders for this test.